### DISCUSSION TITLE
TGCF is not working properly when deployed to render.com

### DISCUSSION THREAD

**Discussion by Lucifer-00007** [aahnik/tgcf#515] [Q&A]
I am trying to deploy tgcf to render.com as web-service. Initially it is running well but after some time the bot stops running.

i am using bot token, and using the free service of render.com

Can anyone help me with this issue?
@nmaiti 
@FloPinguin 
@Matt3oV

**Comment by nmaiti** [comment]
check your 500min/month limit
